src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -267,10 +283,14 @@ public static function createFragment( $html, $context = '<body>', $encoding = '
 	/**
 	 * Constructor.
 	 *
+	 * Do not use this method. Use the static creator methods instead.
+	 *
 	 * @access private
 	 *
 	 * @since 6.4.0
 	 *
+ 	 * @see WP_HTML_Processor::createFragment()
+	 *
```

Comment:
```suggestion
	 * @see WP_HTML_Processor::createFragment()
	 *
```

Fix PHPCS error.

Reply:
Thanks! Removed extra space in [27c3b37](https://github.com/WordPress/wordpress-develop/pull/5126/commits/27c3b3710432d2ad2e0bf2fe3c0bcb301befa6fd)